iron-meta.html

```diff
@@ -284,14 +284,16 @@
       /**
        * Actually a factory method, not a true constructor. Only runs if
        * someone invokes it directly (via `new Polymer.IronMeta()`);
+       *
+       * @param {{type: (string|undefined), key: (string|undefined)}=} opt_config
```

Comment:
This is the only really important line, right? I know the Google style guide has opt_* for optional params, but I don't think that's a general requirement in Polymer code. I think it's fine either way, but @rictic, do you care?